src/components/CMenu/CMenu.js

```diff
@@ -54,7 +54,8 @@ export default {
     },
     setItems() {
       const pages = this.getBindingValue('=$app.pages');
-      this.items = map(pages, page => ({
+      const items = filter(pages, 'showInMenu');
```

Comment:
Currently this is not working inside deployed apps (filter returns null). @Lanchi I believe this requires some changes inside api / deployer, since deployed applications don't have info about showInMenu option

Reply:
Yup, my bad, I forgot about that completely.